docs/contributing/blog-and-website-contributions.md

```diff
@@ -58,6 +58,7 @@ If you want to make changes, improvements, or add new functionality to the websi
 
 - Clone [the Gatsby repo](https://github.com/gatsbyjs/gatsby/) and navigate to `/www`
 - Run `yarn` to install all of the website's dependencies.
+- If you are experiencing issues on a Linux machine, run `sudo apt install libvips-dev`, to install a native dependency. You can also reference [Gatsby guide on Linux](/docs/gatsby-on-linux/) for other Linux-specific requirements.
```

Comment:
Maybe this works better as a foot note as opposed to a point in the regular instructions?

Reply:
Do you mean that I should use link

- Run `yarn` to install all of the website's dependencies.<sup id="linuxdependency">[1](#libvips)</sup>

and add it at the footer like the sample below?

<a id="libvips"><b>1</b></a>: If you are experiencing issues on a Linux machine, run `sudo apt install libvips-dev`, to install a native dependency. You can also reference [Gatsby guide on Linux](/docs/gatsby-on-linux/) for other Linux-specific requirements. [↩](#linuxdependency)

Reply:
I think you could put this item on its own line and start it with `> Note:`
That would be consistent with other notes in the docs.

Reply:
@marcysutton Can you help review my PR. It has been updated.

Reply:
The command given is only valid if you're using `apt` as the package manager(Ubuntu/Debian), on other distros it won't work.

Sharp pulls in linux binaries that should work without needing this(unless you have musl like on Alpine Linux), iirc you need to be using a compatible version of Node for the binaries though(v8 and v10 I think, see the docs for sharp, it has a section about this). If no compatible binary is available, sharp then attempts to compile. **EDIT:** [Here's those docs](https://sharp.dimens.io/en/stable/install/), it's better to link there for troubleshooting.